barretenberg/cpp/scripts/rebuild_avm.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Rebuild
+./scripts/compile_avm.sh
+
+# Format generated folders
+git add **/generated/*
+./format.sh staged
+
+# Build vm tests
+cmake --build --preset clang16 --target vm_tests
```

Comment:
Not sure this (tests) should be here, or even if this script(s) is needed. But don't mind!

Reply:
really just for convinience, but yes not required